packages/atlas/src/providers/nftActions/provider.tsx

```diff
@@ -8,19 +9,18 @@ type ContextValue = {
 
   setCurrentAction: React.Dispatch<React.SetStateAction<NftAction | null>>
   setCurrentNftId: React.Dispatch<React.SetStateAction<string | null>>
+  closeNftAction: () => void
 }
 
 export const NftActionsContext = React.createContext<ContextValue | undefined>(undefined)
 NftActionsContext.displayName = 'NftActionsContext'
 
-type NftAction = 'putOnSale' | 'purchase' | 'settle'
+type NftAction = 'putOnSale' | 'purchase' | 'settle' | 'accept-bid' | 'change-price'
 
 export const NftActionsProvider: React.FC = ({ children }) => {
   const [currentAction, setCurrentAction] = useState<NftAction | null>(null)
   const [currentNftId, setCurrentNftId] = useState<string | null>(null)
 
-  // TODO: remove following code once NftPurchaseView uses BottomDrawer
-  // --START--
```

Comment:
Wait, we shouldn't remove this. This block(with useEffect and state) will be removed in #2310.